src/utils.js

```diff
@@ -46,5 +46,19 @@ module.exports = {
       doc._rev = rev;
       return doc;
     });
+  },
+  strictBulkDocs: docs => {
+    return DB.app.bulkDocs(docs)
+      .then(result => {
+        const errors = result.filter(r => r.error);
+
+        if (errors.length) {
+          const error = Error('bulkDocs did not complete successfully');
+          error.errors = errors;
```

Comment:
Here it looks like `error.errors` is actually an array of `results`.  I guess that's useful, as it likely gives some context to each errored result, but perhaps the property could have a better name?  Otherwise, to reference the first error, you'd write `error.errors[0].error`, which seems weird.

Reply:
Going from the PouchDB api, these things will look like:
```

Reply:
It's a shame there's no indication of which document or revs were in conflict :-\